packages/console/src/mdx-components-v2/Steps/index.tsx

```diff
@@ -29,7 +30,7 @@ const findScrollableElement = (element: Nullable<HTMLElement>): Nullable<HTMLEle
   return findScrollableElement(element.parentElement);
 };
 
-export default function Steps({ children: reactChildren }: Props) {
+export default function Steps({ hasNavigationAnchors = true, children: reactChildren }: Props) {
```

Comment:
no need to pass prop here. you can use `GuideContext`

Reply:
Ah, good point